README.md

```diff
@@ -119,7 +119,7 @@ Pattern/files that will be ignored during linting. Can also be defined in `ui5li
 
 **Example:**
 ```sh
-ui5lint --ignore-pattern "webapp/thirdparty/**"
+ui5lint --ignore-pattern "webapp/thirdparty/**" "webapp/test/**"
```

Comment:
This one is also valid:
`ui5lint --ignore-pattern "webapp/thirdparty/**" --ignore-pattern "webapp/test/**"

Reply:
Does it make sense to have this in the readme as well?

Reply:
I think adding another `--ignore-pattern` is only useful when generically adding options to a commend, e.g. when calling the CLI programatically. When using the option directly, it's easier to do it as suggested in this PR, so I think we should only document that one.

Side-note: I'd suggest to use `"webapp/test/e2e/**"` as an example pattern, as test should be linted, but there may be end-to-end tests e.g. in Node.js (wdio).

Reply:
> Side-note: I'd suggest to use "webapp/test/e2e/**"

changed it